docs/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
+Documenter = "= 0.27.5"
```

Comment:
A workaround for

```
┌ Error: Failed to push:
│   exception =
│    BoundsError: attempt to access 0-element Vector{String} at index [1]
│    Stacktrace:
│      [1] getindex
│        @ ./array.jl:801 [inlined]
│      [2] first
│        @ ./abstractarray.jl:368 [inlined]
│      [3] #74
│        @ ~/.julia/packages/Documenter/ruzxx/src/Writers/HTMLWriter.jl:1422 [inlined]
│      [4] open(::Documenter.Writers.HTMLWriter.var"#74#75"{Vector{String}, String}, ::String, ::Vararg{String, N} where N; kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
│        @ Base ./io.jl:330
│      [5] open
│        @ ./io.jl:328 [inlined]
│      [6] generate_redirect_file(redirectfile::String, entries::Vector{String})
│        @ Documenter.Writers.HTMLWriter ~/.julia/packages/Documenter/ruzxx/src/Writers/HTMLWriter.jl:1420
```

https://github.com/JuliaLang/PublicAPI.jl/runs/3706561146#step:5:61

ref: https://github.com/JuliaDocs/Documenter.jl/commit/cacf739b425b088e285af14a214cf4865bfb0019#r56076137